src/communication/azext_communication/version.py

```diff
@@ -4,7 +4,7 @@
 # --------------------------------------------------------------------------------------------
 
 
-VERSION = '1.5.2'
+VERSION = '1.5.3'
```

Comment:
I think the changes are not that minor, I recommend bumping to 1.6

Reply:
bumped to 1.6